### PR TITLE
Bug/6.3.1/tdi 37860

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-37860

### CODE REVIEW

**Comment by jzhao-talend** [comment]
https://github.com/Talend/tdi-studio-se/blob/aa6c7b4f1edd81d5458517b265cd407ad1adf563/main/plugins/org.talend.designer.components.localprovider/pom.xml#L84
@pyzhou-talend This one should also be changed.
Else the product build would be fails

**Comment by jzhao-talend** [comment]
Please also take care the commit message format.

**Comment by pyzhou-talend** [comment]
I left one method behind (getMultipleLeads), cause I changed this method in another branch. I will fix this bug in that branch in case there is chance to have conflict.

File: main/plugins/org.talend.designer.components.localprovider/components/tMarketoOutputWrite/tMarketoOutputWrite_java.xml [hunk 1]
```diff
@@ -354,11 +354,11 @@
...
 				REQUIRED_IF="(USE_REST_API == 'true')" />
-			<IMPORT
-				NAME="TALEND-MARKETO-REST"
-				MODULE="talend-marketo-rest-2.0-20160622.jar"
-				MVN="mvn:org.talend.libraries/talend-marketo-rest-2.0-20160622/6.2.1"
-				REQUIRED_IF="(USE_REST_API == 'true')" />
+		    <IMPORT 
+		        NAME="TALEND-MARKETO-REST" 
+		        MODULE="talend-marketo-rest-2.1-20161125.jar" 
+		        MVN="mvn:org.talend.libraries/talend-marketo-rest/2.1-20161125"
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMarketoOutputWrite/tMarketoOutputWrite_java.xml]: Sorry. I didn't find in last code review.
Why "/6.2.1" is removed ?

**Pull request merged**